watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -2102,17 +2218,17 @@ def calculate_scaling_factors(self):
         if self.is_property_constructed("elec_cond_phase"):
             if iscale.get_scaling_factor(self.elec_cond_phase) is None:
                 for ind, v in self.elec_cond_phase.items():
-                    sf = (
-                        iscale.get_scaling_factor(self.equiv_conductivity_phase[ind])
-                        * sum(
-                            iscale.get_scaling_factor(
-                                self.conc_mol_phase_comp["Liq", j]
-                            )
-                            ** 2
+                    sf_equiv_cond_phase = iscale.get_scaling_factor(
+                        self.equiv_conductivity_phase[ind]
+                    )
+                    sf_conc_mol_z = value(
+                        1
+                        / sum(
+                            self.conc_mol_phase_comp["Liq", j] * self.charge_comp[j]
                             for j in self.params.cation_set
                         )
-                        ** 0.5
                     )
+                    sf = sf_equiv_cond_phase * sf_conc_mol_z
```

Comment:
Is there evidence for the updated scaling method being better than the original ?

Reply:
I recall that the reason I changed this was because in my new tests, I included an assertion that no badly scaled variables occurred. From my recollection, at least one of these properties related to conductivity popped up as "badly scaled" (not by much though). So I altered the scaling and this improved the scaled value and remove said property from the list of badly scaled properties. I may have just gone on to apply a similar change to the other property related to conductivity just for consistency. 

Additionally, since you asked, I assumed that the ED flowsheet would be the first thing to check, assuming it uses these properties. I ran the feed/bleed flowsheet and the 1D flowsheet; on main (without my changes) just looking to the number of iterations for the solves where tee=True (three solves in feed/bleed with tee=True), I obtained 22, 71, and 27 iterations; in comparison, on this branch, I obtained 19, 56, and 27 iterations, so slight reduction in iterations at least for the first two solves with tee=True. For 1D ED flowsheet, I see a reduction from 25 to 23 iterations with my changes, so not critical, but still less. At the very least, these changes don't seem to affect anything in a negative manner.